config/initializers/new_framework_defaults.rb

```diff
@@ -0,0 +1,23 @@
+# Be sure to restart your server when you modify this file.
+#
+# This file contains migration options to ease your Rails 5.0 upgrade.
+#
+# Read the Rails 5.0 release notes for more info on each option.
+
+# TODO Rails5Upgrade - Once upgraded flip defaults one by one to migrate to the new default.
+
+# Enable per-form CSRF tokens. Previous versions had false.
+Rails.application.config.action_controller.per_form_csrf_tokens = false
+
+# Enable origin-checking CSRF mitigation. Previous versions had false.
+Rails.application.config.action_controller.forgery_protection_origin_check = false
+
+# Make Ruby 2.4 preserve the timezone of the receiver when calling `to_time`.
+# Previous versions had false.
+ActiveSupport.to_time_preserves_timezone = false
+
+# Require `belongs_to` associations by default. Previous versions had false.
+Rails.application.config.active_record.belongs_to_required_by_default = false
+
+# Do not halt callback chains when a callback returns false. Previous versions had true.
+ActiveSupport.halt_callback_chains_on_return_false = true
```

Comment:
This is left intact to leave feedback application configured the same way as before. Once upgrade has happened, it's up to team who owns it to modify it.

Reply:
This was auto-generated file by rails 5 update script. If you are referring to lines 22/23 - ActiveSupport.halt_callback_chains_on_return_false = true means to halt callback chains when a callback returns false, which is the old behavior.